client/js/lounge.js

```diff
@@ -840,7 +847,6 @@ $(function() {
 		self.addClass("active")
 			.find(".badge")
 			.removeClass("highlight")
-			.data("count", 0)
```

Comment:
There's no longer a need for it, as it only ever updates the badge from the
server now.